benches/oneshot/ipa.rs

```diff
@@ -117,10 +133,11 @@ async fn run(args: Args) -> Result<(), Error> {
             max_trigger_value: NonZeroU32::try_from(args.max_trigger_value).unwrap(),
             max_breakdown_key: NonZeroU32::try_from(args.breakdown_keys).unwrap(),
             max_events_per_user: NonZeroU32::try_from(args.records_per_user).unwrap(),
+            single_user_mode,
             ..Default::default()
         },
     )
-    .take(args.query_size)
+    .take(query_size)
```

Comment:
This seems wrong... If you're running things in OPRF mode, and you specify a query_size, it will now be totally ignored? I think we want to use the oneshot test for more than just step generation, right?

Reply:
This is only the case when `cfg!(feature = "step-trace")` is true, so all other tests will be run as usual.